src/main/java/utils/WaitHelper.java

```diff
@@ -1,28 +1,29 @@
-package wrapper;
+package utils;
 
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
+import webdriver.DriverWrapper;
 
 import java.util.concurrent.TimeUnit;
 
-public class WaitWrapper {
+public class WaitHelper {
 
     public static void waitPageLoad(){
-        DriverWrapper.getDriver().manage().timeouts().pageLoadTimeout(5, TimeUnit.SECONDS);
+        DriverWrapper.getDriver().manage().timeouts().pageLoadTimeout(20, TimeUnit.SECONDS);
     }
 
     public static void implicitlyWait(){
-        DriverWrapper.getDriver().manage().timeouts().implicitlyWait(3,TimeUnit.SECONDS);
+        DriverWrapper.getDriver().manage().timeouts().implicitlyWait(10,TimeUnit.SECONDS);
     }
 
     public static WebElement waitForElementVisible(WebElement element){
-        new WebDriverWait(DriverWrapper.getDriver(),10).until(ExpectedConditions.visibilityOf(element));
+        new WebDriverWait(DriverWrapper.getDriver(),20).until(ExpectedConditions.visibilityOf(element));
         return element;
     }
 
     public static WebElement waitForElementClickable(WebElement element){
-        new WebDriverWait(DriverWrapper.getDriver(), 10).until(ExpectedConditions.elementToBeClickable(element));
+        new WebDriverWait(DriverWrapper.getDriver(), 20).until(ExpectedConditions.elementToBeClickable(element));
```

Comment:
move time out time to constant to make it easier to change if you will need more timeout